apps/Learning-Buddy/screens/LoginScreen.jsx

```diff
@@ -29,7 +29,7 @@ export const LoginScreen = ({ navigation }) => {
     if (formVerify()) {
       const status = await signInWithEmail(email, password);
       if (status === 'SignedIn') {
-        Alert.alert('Success! You Are Signed In!');
+        // Alert.alert('Success! You Are Signed In!');
```

Comment:
You can just straight up delete this if you want, no need to save it as a comment (if someone needs something like this for testing down the line, they can re-add it later.) We want to keep our code clean and comment-free wherever possible :)

Reply:
I like the idea to notify on the login success but it is only the pop-up that I find annoying.
I am planning to change the notification to close itself.